app/src/main/java/org/eyeseetea/malariacare/domain/usecase/GetSurveyAnsweredRatioUseCase.java

```diff
@@ -9,11 +9,6 @@
 
 public class GetSurveyAnsweredRatioUseCase{
```

Comment:
UseCases should use AsynExecutor to execute and MainExecutor to invoke callbacks